go.mod

```diff
@@ -1,6 +1,6 @@
-module marwan.io/otel-exporter-datadog
+module github.com/chrisleavoy/otel-exporter-datadog
 
-go 1.14
+go 1.13
```

Comment:
let's stick with `go 1.14`

Reply:
any reason why? most things are still pinned to 1.13.  i haven't been writing go in awhile but I've notice go mods are pretty annoying right now

Reply:
This pins any module-features to ensure backwards compatibility, so when it comes to vendoring and resolution, we want to keep the behavior to what it was originally intended. 

If you have a server in 1.13, that wants to import this and cannot, then this acts as an encouragement to upgrade your server to the latest Go distro :)

Reply:
Well I think I’m stuck on 1.13 for now.

Reply:
Well I think I’m stuck on 1.13 for now.

Reply:
Cool cool, let's keep this 1.13 then, although I don't see why you're unable change your own `go.mod` from `1.13` to `1.14` :)